core/common/src/main/java/alluxio/Constants.java

```diff
@@ -160,6 +161,12 @@
   public static final String UNDERFS_S3_ADMIN_THREADS_MAX = "alluxio.underfs.s3.admin.threads.max";
   public static final String UNDERFS_S3_UPLOAD_THREADS_MAX =
       "alluxio.underfs.s3.upload.threads.max";
+  public static final String UNDERFS_S3A_SECURE_HTTP_ENABLED =
+      "alluxio.underfs.s3a.secure.http.enabled";
+  public static final String UNDERFS_S3A_SERVER_SIDE_ENCRYPTION_ENABLED =
```

Comment:
Yeah, I think ideally we should have most parameters with `s3`, which work for both `s3n` and `s3a`.
`s3a` parameters should be minimal and only for those are not supported in `s3n`